### PR TITLE
More Filters:  UI for multiple contributors filters 

### LINKED ISSUE #6065: Remove exposure of sierraId and lang params
**What**

The `sierraId` and `lang` parameters are used to fetch a work's item. These should never be exposed, and should definitely not be relied upon.

**Where**

All `item` pages

https://wellcomecollection.org/works/peu4g523/items?sierraId=b20452019&langCode=per&canvas=1

Should be 
https://wellcomecollection.org/works/peu4g523/items?canvas=1

### PR DESCRIPTION
**Pull request by ishouty**
UI multiple contributors for desktop and mobile
Non Javascript
Reset Active Filters update

closes #6065

### CODE REVIEW

File: catalogue/webapp/pages/works.tsx [hunk 1]
```diff
@@ -289,8 +289,12 @@ export const getServerSideProps: GetServerSideProps<
   const params = WorksRoute.fromQuery(parsedParams);
   const { searchMoreFilters } = globalContextData.toggles;
   const defaultAggregations = ['workType', 'locationType'];
-
-  const moreFiltersAggregations = ['genres', 'languages', 'subjects'];
+  const moreFiltersAggregations = [
```
Comment (jamesgorrie) [inline catalogue/webapp/pages/works.tsx:292]: Can we just have this in the aggregations list?

File: common/model/catalogue.ts [hunk 2]
```diff
@@ -263,6 +263,22 @@ export type CatalogueAggregationBucket = {
...
 };
 
+export type CatalogueAggregationContributorsBucket = {
+  count: number;
+  data: {
+    agent: {
+      label: string;
+      type: string;
+    };
+    roles: string[];
```
Comment (jamesgorrie) [inline common/model/catalogue.ts:273]: Let's ignore this for now as it's not actually a list of string, but a list of roles, but unnecessary.

File: catalogue/webapp/components/WorkDetails/WorkDetails.tsx [hunk 3]
```diff
@@ -462,6 +462,7 @@ const WorkDetails: FunctionComponent<Props> = ({ work, itemUrl }: Props) => {
                 languages: [],
                 subjectsLabel: null,
                 genresLabel: null,
+                contributorsLabel: null,
```
Comment (jamesgorrie) [inline catalogue/webapp/components/WorkDetails/WorkDetails.tsx:465]: Just for consistency - could we please call this
```suggestion
                contributorsAgentLabel: null,
```

File: common/utils/filters.ts [hunk 4]
```diff
@@ -35,6 +36,14 @@ export const getAggregationFilterByName = (
     : [];
 };
 
+export const getAggregationContributors = (
+  aggregations: CatalogueAggregations | undefined
+): CatalogueAggregationContributorsBucket[] | [] => {
+  return aggregations && aggregations?.['contributors']?.buckets
+    ? aggregations?.['contributors']?.buckets
+    : [];
+};
```
Comment (jamesgorrie) [inline common/utils/filters.ts:45]: What's the difference between this and `getAggregationByName(aggs, 'contributors')`?

**Review by jamesgorrie — APPROVED** [review]

File: common/utils/filters.ts (hunk 4 above)
Comment (ishouty) [reply common/utils/filters.ts:45]: i was having issues with the return type so i had to make its own one CatalogueAggregationContributorsBucket[]. the object type return is different.

Comment (jamesgorrie) [reply common/utils/filters.ts:45]: Yea, this is actually something I would like to address in the API - so I can use this as an example.

Makes sense to leave this then.

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**